jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaLineMarker.kt

```diff
@@ -37,25 +42,28 @@ class LambdaLineMarker : LineMarkerProviderDescriptor() {
         } ?: return null
 
         val handler = handlerResolver.determineHandler(element) ?: return null
+        val runtime = findModuleForPsiElement(element)?.let { RuntimeGroup.determineRuntime(it) } ?: return null
 
-        val actionGroup = DefaultActionGroup()
+        return if (isHandlerValid(element, handler, runtime)) {
+            val actionGroup = DefaultActionGroup()
 
-        if (element.language in LambdaPackager.supportedLanguages) {
-            val executorActions = ExecutorAction.getActions(1)
-            executorActions.forEach {
-                actionGroup.add(LineMarkerActionWrapper(element, it))
-            }
+            if (element.language in LambdaPackager.supportedLanguages) {
```

Comment:
Something weird going on with your spacing here - can you check that you're using the same style conventions?